samples/drivers/flash_shell/src/main.c

```diff
@@ -579,9 +579,11 @@ static int cmd_set_dev(const struct shell *shell, size_t argc, char **argv)
 
 void main(void)
 {
-	flash_device = device_get_binding(DT_FLASH_DEV_NAME);
+	flash_device =
+		device_get_binding(DT_CHOSEN_ZEPHYR_FLASH_CTRL_LABEL);
 	if (flash_device) {
-		printk("Found flash device %s.\n", DT_FLASH_DEV_NAME);
+		printk("Found flash device %s.\n",
```

Comment:
```suggestion
		printk("Found flash controller %s.\n",
```

Just a hint to developers that something changed.

Reply:
Done.